flink-table/flink-table-common/src/main/java/org/apache/flink/table/catalog/ReadableCatalog.java

```diff
@@ -132,4 +133,36 @@
 	 */
 	boolean tableExists(ObjectPath objectPath) throws CatalogException;
 
+	// ------ functions ------
+
+	/**
+	 * List the names of all functions in the given database. An empty list is returned if none is registered.
+	 *
+	 * @param dbName	Name of the database.
```

Comment:
replace tab with space, also should use lower case for starting character, like "name of the database"

all javadoc in this file needs some reformatting

Reply:
I prefer reformatting happens in a separate PR as it would otherwise pollute the review here. However, I have reformatted the newly added methods.